### PR TITLE
Display gem image + change header background color in Carousel

### PR DESCRIPTION
**Pull request by sebastiantoh**
TODO:
- implement attachment uploads and integrate with backend
  - for attachments, do we want separate buttons for gallery and camera? Because I'm planning to use this (https://youzan.github.io/vant/#/en-US/uploader) and for my phone, tapping on the button will bring up a prompt suggesting either gallery / camera, so I'm not sure if it's repetitive. Might just be my phone tho

image

### CODE REVIEW

File: frontend/src/components/GemHeader.vue [hunk 1]
```diff
@@ -1,45 +1,61 @@
...
-.square {
-  height: 250px;
-  width: 100%;
-  background-color: #e4bfbf;
+  .semi-circle {
+    transform: translate(-25%, -50%);
+    height: 80vh;
+    width: 200vw;
+    border-radius: 0 0 50% 50%;
+    background-color: #e4bfbf;
```
Comment (ming-00) [inline frontend/src/components/GemHeader.vue:57]: Maybe we can also add a transition: background-color 1s ease; here to make the bg transition less abrupt?

**Comment by ming-00** [comment]
Looks great 😻 

>   * for attachments, do we want separate buttons for gallery and camera? Because I'm planning to use this (https://youzan.github.io/vant/#/en-US/uploader) and for my phone, tapping on the button will bring up a prompt suggesting either gallery / camera, so I'm not sure if it's repetitive. Might just be my phone tho

Ooh sure, let's just go with a single button then?

File: frontend/src/components/GemHeader.vue (hunk 1 above)
Comment (sebastiantoh) [reply frontend/src/components/GemHeader.vue:57]: oh, nice! added!

**Review by ming-00 — APPROVED** [review]
LGTM~

**Pull request merged**